components/dialog/dialog-mixin.js

```diff
@@ -416,8 +415,7 @@ export const DialogMixin = superclass => class extends RtlMixin(superclass) {
 			return node.classList && node.classList.contains('d2l-dialog-outer');
 		});
 
-		// native dialog backdrop does not prevent body scrolling
```

Comment:
@dbatiste I guess this comment, which I read many times, should have tipped me off 😆